cypress/e2e/auth/auth.cy.js

```diff
@@ -0,0 +1,138 @@
+    loginPagePO.clickLogin();
+    loginPagePO.elements.loginBtn().should("be.disabled");
+  });
+
+  it("Verify the validation msg when login is attempted with less than 6 letter password", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString(3);
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+  loginPagePO.verifyPasswordLengthErrorMessage();
+
+   
+  });
+  it("Verify the login process with invalidLogin credentials ", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString(7);
+    // cy.log(randomGenerateName)
+    // cy.pause()
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+    loginPagePO.verifyInvalidCredentialAlertMessage();
+  });
+  it("should visible the password", function () {
+    cy.login();
+    cy.get("#showPassword").click();
+    cy.get("#password").invoke("attr", "type").should("eq", "text");
+  });
+  it("It should not be submit with empty email address", function () {
+    const randomGenerateName = getRandomString(6);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+    loginPagePO.elements.loginBtn().should("be.disabled");
+  });
+  it(" should not be submit with empty password", function () {
+    const randomGenerateEmail = getRandomEmail();
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.clickLogin();
+    loginPagePO.elements.loginBtn().should("be.disabled");
+  });
+  it("It should not be submit with incorrect password", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString();
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+    loginPagePO.verifyPasswordLengthErrorMessage();
+  });
+
+  it("It should not be submit with incorrect email address", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString(7);
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+    loginPagePO.verifyInvalidCredentialAlertMessage();
+  });
+
+  it("clicked on forget password redirected to forget password page", function () {
+    cy.get(".forgot-password").click();
+    cy.url().should(
+      "eq",
+      "https://frontendbootcamp.proshore.eu/accounts/password-forgot"
+    );
+  });
+
+  it("URL of reset password  should be https://frontendbootcamp.proshore.eu/accounts/password-forgot", function () {
+    cy.get(".forgot-password").click();
+    cy.url().should(
+      "eq",
+      "https://frontendbootcamp.proshore.eu/accounts/password-forgot"
+    );
+  });
+  it(" should  be submit with empty emailaddress", function () {
```

Comment:
the description should rather be it should not be submitted with empty email address